lib/libc/zoneinfo/mk_latest_tzrule.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
```

Comment:
How about moving this script to TizenRT/tools folder? Most of the tool scripts are in there.